### ISSUE TITLE
Bar labels not centered; option to change bar colors?

### ISSUE THREAD

**Issue by alexeble** [bbdaniels/betterbar#8]
Am trying to generate a bar chart with bar labels listing the value of the mean above the relevant bar. For some reason these labels are off-center to the right, and overlap with the bar itself (see attached picture).

Code used is 

betterbar math lang total, over(intervention) v graphregion(color(white)) ysc(noline) ///
	ylab(0 10 20 30 40 50 60 70 80 90 100, grid notick) ysc(range(0 105)) ///
	yscale(range(0 105)) legend(order(1 "Control" 2 "Intervention")) yti("Test score (out of 100)") bgcolor(white) bar		

Also looking for the code options to change the color of the bars. 

Can fix both in graph editor, but ideally would love to automate. Thanks! 

<img width="1273" alt="Screen Shot 2019-06-13 at 12 04 05 PM" src="https://user-images.githubusercontent.com/12138367/59449068-65ae0f80-8dd4-11e9-8759-f3514a3b82f5.png">

**Comment by bbdaniels** [comment]
This is implemented in c80d38c73f804da4a7df4f867a3be2385ca651c0 and d8c6c303ff02998e374671e0cd995c5ab0c85721 -- This is only available on the `develop` branch until the next release but you should now be able to get the ado file manually!

```
betterbar price mpg, over(foreign) bar v ///
  legend(on) barc(red blue)
```

<img width="588" alt="Screenshot 2019-06-15 20 16 06" src="https://user-images.githubusercontent.com/1459122/59557549-787a3d00-8faa-11e9-80f5-64a0cd548359.png">

**Comment by alexeble** [comment]
Thanks!!!
